cypress/integration/code-schools.spec.js

```diff
@@ -0,0 +1,12 @@
+// Assert that when the user navigates to the page, the code schools are all rendered on the page
+describe('visit page', () => {
```

Comment:
You're off to a good start.

I think you'll find value in checking out the examples of existing tests! I was expecting the assertion to look something like... ensuring that multiple CodeSchoolCards were findable

Try throwing this around all of your tests - this well help you get off to the right start...
```javascript
describe('code schools', function() {
  beforeEach(() => {
    cy.server(); // allows us to begin dealing with HTTP requests
    cy.route('GET', '/api/v1/code_schools').as('codeSchools'); // names the expected API call for usage in `cy.wait()` 

    // custom function defined in `cypress/support/commands.js`
    // DRYing up the reliable way of getting too and waiting for a page to load
    cy.visitAndWaitFor('/code_schools');

    // ensuring the page has been visited before the rest of any test begins
    cy.get('h1').should('have.text', 'Code Schools');
  });

// ... actual tests (each assertion defined in issue #390 - test per bullet)
});
```

Reply:
Great feedback.  I will make some adjustments and re-pull later today.